### PR TITLE
Add Proxmox virtualization guide

### PR DESCRIPTION
**Pull request by Coldfirex**
Adding a guide to virtualize Haiku within Proxmox VE

First pass.  I also havent touched github in years so I more than likely messed something up.

Edit: I need to get Hugo setup first so I can actually verify how this looks.

### CODE REVIEW

**Comment by Coldfirex** [comment]
Got Hugo going and made edits.  Oddly on my local copy of Hugo the article does not correctly display 3 of the images even though I see nothing wrong with them.  Ideas?

File: content/guides/virtualizing/proxmox.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+![](/files/guides/virtualizing/Proxmox/CreateVM-Memory.png)
+
+Click Next to the Network tab.  Select the Model of network interface card (NIC) you would like to use.  The default of VirtIO works.
+![](/files/guides/virtualizing/Proxmox/CreateVM-Network.png)
+
+Click Next to the Confirm tab.  Review the configuration and select Finish to complete the creation of your Haiku VM.
+![](/files/guides/virtualizing/Proxmox/CreateVM-Confirm.png)
+
+After completing the wizard, move to the Virtual Machines list. Your Haiku machine should now be listed. Select it and press `Start`, then click 'Console' button to bring up a window to your VM. If you see the Haiku boot screen, then you configured everything correctly.  You can now proceed to install Haiku or try out the live environment.
+![](/files/guides/virtualizing/Proxmox/StartVM.png}
```
Comment (humdingerb) [inline content/guides/virtualizing/proxmox.md:46]:
```suggestion
![](/files/guides/virtualizing/Proxmox/StartVM.png)
```

File: content/guides/virtualizing/proxmox.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Installing Haiku
+
+Before finally installing Haiku, you need to partition the virtual hard drive. Open the Haiku Installer and click on `Set up partitions…` end.
+
+![](/files/guides/virtualizing/Proxmox/Installer1.png}
```
Comment (humdingerb) [inline content/guides/virtualizing/proxmox.md:52]:
```suggestion
![](/files/guides/virtualizing/Proxmox/Installer1.png)
```

File: content/guides/virtualizing/proxmox.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+![](/files/guides/virtualizing/Proxmox/Installer2.png}
```
Comment (humdingerb) [inline content/guides/virtualizing/proxmox.md:53]:
```suggestion
![](/files/guides/virtualizing/Proxmox/Installer2.png)
```

File: content/guides/virtualizing/proxmox.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
++++
+type = "article"
+title = "Emulating Haiku In Proxmox"
```
Comment (humdingerb) [inline content/guides/virtualizing/proxmox.md:3]:
```suggestion
title = "Emulating Haiku in Proxmox"
```

**Review by humdingerb — COMMENTED** [review]
Mislayed your glasses, ey? :)

If you want to, you can use e.g. <span class="button">Next</span> to get css formatted buttons. There's also "key", "menu" and "path".

**Comment by Coldfirex** [comment]
> Mislayed your glasses, ey? :)
> 
> If you want to, you can use e.g. Next to get css formatted buttons. There's also "key", "menu" and "path".

Apparently so!  Thank you!

**Comment by humdingerb** [comment]
> What do you mean about the buttons part?

Sorry, I forgot to escape those pesky pointy brackets. I meant, you could use `<span class="button">`ButtonLabel`</span>`. Same with "menu", "key" and "path" classes.

**Comment by Coldfirex** [comment]
I am dense.  Are you suggesting to add the button label for when I mention to click 'Next' as an example in the instructions?

**Comment by humdingerb** [comment]
Only if you want to. :)
By putting a button's label in `<span class="button"></span>` tags, it'll appear more button-like. See e.g. https://www.haiku-os.org/guides/installing/making_haiku_usb_stick . Same for pressing keys with the "key" class.

**Comment by Coldfirex** [comment]
I attempted but it wasnt showing on my side, so I say just go with it for now, if you are ok with it.

**Comment by humdingerb** [comment]
Sure. Maybe we'll go over all guides some time to insert these niceties.
Thanks for now!

**Comment by humdingerb** [comment]
Oh, I just noticed: You'd need to edit the `_index.md` file to add your new guide.

**Comment by Coldfirex** [comment]
Where is that file?

**Comment by humdingerb** [comment]
It's in the same folder as the guides: `website/content/guides/virtualizing/_index.md`

**Comment by Coldfirex** [comment]
Hows that look?

**Comment by humdingerb** [comment]
Looks good to merge! :)
Thanks!

**Pull request merged**